bash/05_export.sh

```diff
@@ -17,6 +17,8 @@ mkdir -p output && (
     CSV_export cpdb_projects &
     CSV_export cpdb_budgets &
     CSV_export cpdb_projects_spending_byyear &
+    CSV_export ddc_capitalprojects_infrastructure &
```

Comment:
Why are these two tables added to the export?  These two tables should be removed.

Reply:
Ahh sorry these two files were added to help and try to debug the outputs on our local machine vs what was being built via github actions. I will remove them

Reply:
I think it would be valuable to go over the local cpdb outputs and how they differ from the github actions build, any idea as to why this might be?

Reply:
@AmandaDoyle recent push addresses your comment